rest_tools/server/handler.py

```diff
@@ -174,8 +176,9 @@ def get_argument(
         strip: bool = True,
         type_: Optional[type] = None,
```

Comment:
`type_` feels a bit weird to use as part of the API. I know the underscore is to avoid keyword clashes, but it feels like "internal variable brought up to the API level".

Any chance you'd rename it `as_type` or something similar?